examples/platform/efr32/MemMonitoring.cpp

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Can't we make a build target for this, and add the

```
declare_args() {
  # Monitor & log RAM usage at runtime. 
  enable_heap_monitoring = false
}
```

there? To avoid unnecessary duplication.

Reply:
I prefer leaving the duplication as it is right now but we'll keep it in mind for a future PR.